packages/web-twig/src/Resources/components/UNSTABLE_EmptyState/README.md

```diff
@@ -0,0 +1,3 @@
+# UNSTABLE EmptyState
+
+⚠️ This component is UNSTABLE and there could be breaking changes in the future. Please use it with caution.
```

Comment:
Let's polish the form before we seed the message into all unstable components 🙂 

https://github.com/lmc-eu/spirit-design-system/pull/1456#discussion_r1626043831